imports/client/components/Celebration.tsx

```diff
@@ -9,6 +10,32 @@ interface CelebrationProps {
   onClose: () => void;
 }
 
+const CelebrationOverlay = styled.div`
+  position: fixed;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  background-color: rgba(0, 0, 0, .20);
+  z-index: 1050;
+`;
+
+const CelebrationContainer = styled.div`
+  background-color: #f0fff0;
+  padding: 24px;
+`;
+
+const CelebrationClose = styled.button`
```

Comment:
Not super attached to this, but for some reason the first thing I thought was "this makes it harder to tell that the element is actually a `<button>`, which makes it slightly harder to understand what props it's likely to accept" -- maybe call this `CelebrationCloseButton`, which hints at its underlying control?